src/uag.c

```diff
@@ -12,23 +12,23 @@
 /* One instance */
 
 static struct uag uag = {
-	NULL,
-	LIST_INIT,
-	NULL,
-	NULL,
-	NULL,
-	NULL,
-	false,
-	0,
-	NULL,
-	NULL,
-	false,
-	false,
-	NULL,
-	NULL,
+	NULL,         /**< cfg               */
+	LIST_INIT,    /**< ual               */
+	NULL,         /**< sip               */
+	NULL,         /**< lsnr              */
+	NULL,         /**< sock              */
+	NULL,         /**< evsock            */
+	0,            /**< transports        */
+	false,        /**< delayed_close     */
+	NULL,         /**< subh              */
+	NULL,         /**< exith             */
+	false,        /**< nodial            */
+	false,        /**< dnd               */
+	NULL,         /**< arg               */
+	NULL,         /**< eprm              */
 #ifdef USE_TLS
-	NULL,
-	NULL
+	NULL,         /**< tls               */
+	NULL          /**< wss_tls           */
```

Comment:
Should we replace this by `memset(&uag, 0, sizeof(uag))` in `ua_init()`?

Reply:
You can replace this by:

`static struct uag uag = {.ual = LIST_INIT};`

All none specified members are initialized by 0 or similar zero type value.